### ISSUE TITLE
Broken with ffmpeg 7.0

### ISSUE THREAD

**Issue by lovesegfault** [nathanbabcock/ffmpeg-sidecar#31]
I have been successfully using this crate for a few months on a personal project, but when I tried to update to `ffmpeg` to the new 7.0 release today, I ran into some issues.

In particular, it seems `FfmpegEvent::Progress` no longer parses correctly, causing that event to never be processed.

**Comment by nathanbabcock** [comment]
Ooh, thanks for flagging this. I know 7.0 introduced multithreading in the FFmpeg CLI. I wonder if that will require any changes to this library, or if it's something smaller like a different format for progress messages.

I'll look into this — PRs welcome in the mean time.

**Comment by nathanbabcock** [comment]
Good news. It's an incredibly minor formatting change in the `Lsize` field of the progress message.

Before (6.0 and earlier):
```
[info] frame= 1996 fps=1984 q=-1.0 Lsize=     372kB time=00:01:19.72 bitrate=  38.2kbits/s speed=79.2x\n
```

After (7.0 onwards):

```
[info] frame=    5 fps=0.0 q=-1.0 Lsize=      10KiB time=00:00:03.00 bitrate=  27.2kbits/s speed= 283x\n
```

**TL;DR:** `kB` changed to `KiB` 😅

**Comment by avsaase** [comment]
Did ffmpeg always report the size in KiB and erroneously report it as KB? Or did they actually change the unit of measurement?

**Comment by nathanbabcock** [comment]
@avsaase that's a really good question. My assumption was that they were using KiB all along, but had a nonstandard abbreviation for it ("kB" with lowercase "k"). However I can't find any concrete evidence in the release notes to verify that.

**Comment by lovesegfault** [comment]
Awesome, thank you so much @nathanbabcock!

Looking forward to the next release 🎉

**Comment by nathanbabcock** [comment]
@avsaase Here's something relevant in the main page of the docs under ["vstats file format"](https://ffmpeg.org/ffmpeg.html#vstats-file-format):

They give an example readout:

```
frame= FRAME q= FRAME_QUALITY PSNR= PSNR f_size= FRAME_SIZE s_size= STREAM_SIZEkB time= TIMESTAMP br= BITRATEkbits/s avg_br= AVERAGE_BITRATEkbits/s
```

Note the `s_size` with the old "kB" suffix. However when they explain each parameter in more detail they clarify:

> `s_size`: stream size expressed in KiB

So I do think it was an ambiguity in their old labelling format, and now they've adopted a more universal abbreviation "KiB" to make it more clear.

---

@lovesegfault Thank you for reporting this issue! I'll publish a release later today or this weekend. Let me know if you spot any other regressions with 7.0.

**Comment by lovesegfault** [comment]
Just tested the patch, and everything is working beautifully now.

Also, somewhere between svtav1 2.0 and ffmpeg 7.0 my transcode fps has **doubled** 🥳
